tools/block-generator/data/conduit.yml

```diff
@@ -0,0 +1,61 @@
+# Log verbosity: PANIC, FATAL, ERROR, WARN, INFO, DEBUG, TRACE
+log-level: {{.LogLevel}}
```

Comment:
conflicted nit (because you lose the syntax highlighting): consider renaming to `conduit.yml.tmpl`